ci-operator/jobs/rhobs/observability-operator/rhobs-observability-operator-main-periodics.yaml

```diff
@@ -10,34 +10,24 @@ periodics:
     org: rhobs
     repo: observability-operator
   labels:
-    ci-operator.openshift.io/variant: periodic
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-qe
+    ci-operator.openshift.io/variant: ocp
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-rhobs-observability-operator-main-periodic-e2e-test-harness
-  reporter_config:
-    slack:
-      channel: '#observability-operator-cicd'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
-        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> {{end}}'
```

Comment:
can you please re-add the reporter config? That has to be done in the generated job config...normally that shouldn't change. See https://docs.ci.openshift.org/docs/how-tos/notification/

Maybe that's why we went with the `periodic` variant initially?

What is the benefit of this change?

Reply:
To integrate with testgrid, ocp the key word is needed